src/JustBehave/JustBehave.csproj

```diff
@@ -1,19 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <Import Project="..\JustBehave.Common.props" />
   <PropertyGroup>
-    <TargetFrameworks>net452</TargetFrameworks>
-  </PropertyGroup>
-  <PropertyGroup>
-    <DefineConstants>TRACE;CODE_ANALYSIS;</DefineConstants>
+    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <NoWarn>$(NoWarn);NU1701</NoWarn>
```

Comment:
What does `NU1701` correspond to?

Reply:
It's the warning that you are using a .NET framework assembly targeting netstandard/netcoreapp and things could go wrong at runtime. It might not be needed now that RhinoMocks is gone, I'll check.

Reply:
Removed as no longer required 👍